digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/client/CacheClient.scala

```diff
@@ -24,7 +24,9 @@ class CacheClient {
 
   def set[DataModel](key: String, ttl: Int, data: DataModel): DataModel = {
     try {
-      client.set(key, ttl, data).isDone
+      LogUtils.time(logger,"Cache value")(
+        client.set(key, ttl, data).isDone
+      )
       client.get(key).asInstanceOf[DataModel]
```

Comment:
Nyt vasta huomasin tän. Tämähän on täysin turha haku datalle joka on just sinne laitettu. Eli palauta suoraan tuo parametrina annettu data, jos setterin nyt täytyy mitään palauttaa muutenkaan...

Reply:
kyl hän se jotakin pitää palautta muuten ketju menee poikki, se tallentaa mutta ei palauta seuraavalle metodille mitään, Hyym tietty velvollisuus rajoja miettien se ei sinäänsä kuulu tähän kohtaan

Reply:
Setterit yleensä ei palauta mitään. Kutsujallahan on hallussaan jo se tieto mitä sinne on asetettu, jos poikkeusta ei tule.

Reply:
![image](https://user-images.githubusercontent.com/70522354/129517246-9148b2e1-b85f-4112-851e-65135c444323.png)
olisko näin oikeampi sitten

Reply:
ei

Reply:
Jos sen jotain pitää palauttaa niin sitten noin.